xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -235,6 +235,9 @@ void CXBMCApp::onStart()
     intentFilter.addAction("android.net.conn.CONNECTIVITY_CHANGE");
     registerReceiver(*this, intentFilter);
     m_mediaSession.reset(new CJNIXBMCMediaSession());
+    m_activityManager.reset(new CJNIActivityManager(getSystemService(CJNIContext::ACTIVITY_SERVICE)));
+    m_inputHandler.setDPI(GetDPI());
+    RegisterDisplayListener();
```

Comment:
Is onStart the correct place?

reading https://developer.android.com/reference/android/app/Activity#onStart() makes it sound like it can be called multiple times (after onrestart and oncreate). 
whereas onDestroy i think is the correct place.

Keep in mind im not across the call stack (java <-> c interactions), so maybe im wrong.

Reply:
actually, i guess nevermind. theres a m_firstrun bool that "protects" multiple calls.
i wonder why that wasnt done in an onCreate() function. anyways, all good, not your issue.

Reply:
because NDK does not call onCreate, for whatever reason.

Reply:
https://developer.android.com/reference/games/game-activity/group/android-native-app-glue

There is APP_CMD_DESTROY, and APP_CMD_START, but no APP_CMD_CREATE